pom.xml

```diff
@@ -660,5 +658,28 @@
       </dependencies>
 
     </profile>
+    <profile>
+      <id>dev-clean</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-clean-plugin</artifactId>
+            <version>2.6.1</version>
+            <configuration>
+              <excludeDefaultDirectories>true</excludeDefaultDirectories>
+              <filesets>
+                <fileset>
+                  <directory>node_modules</directory>
+                  <excludes>
+                    <exclude>**</exclude>
```

Comment:
Hm Are you sure it deletes anything with this settings? You excluded the defaults and I don't see any includes

Reply:
You are right. I noticed that the clean without profile will delete the node_modules folder. So In the dev-clean profile, I removed the part that explicitly deletes node_modules

Reply:
But are you sure it deletes other stuff? E.g. if target directory is present, does it delete it? Looking at the setting I don't see this settings deleting anything.

Reply:
Yea I tested it. It will delete target/